src/preprocess/etl.py

```diff
@@ -212,18 +212,20 @@ def preprocess(df, target_col: str = 'churn'):
 
 
 class DataPreprocess:
-    def __init__(self, data_path_parent_level: int = 2):
+    def __init__(self, contains_str: str = 'dataset', n_cols: int = 21, data_path_parent_level: int = 2):
         self.data_path_parent_level = data_path_parent_level
+        self.contains_str = contains_str
+        self.n_cols = n_cols
 
     def load_and_encode_data(self):
         # the parent directory of input data path
         input_data_path = pl.Path(__file__).resolve().parents[self.data_path_parent_level].joinpath('data/input/')
         check_data = CheckRawInputs(input_data_path)
```

Comment:
I feel if it is a class, you should call it RawInputsChecker or RawInputsProcessor, etc

Reply:
updated!